AeroGearPush-Sw.podspec

```diff
@@ -1,11 +1,11 @@
 Pod::Spec.new do |s|
-  s.name         = "AeroGearPush-Swift"
-  s.version      = "3.0.0"
+  s.name         = "AeroGearPush-Sw"
```

Comment:
Why the project was renamed?

Reply:
@danielpassos  There are breaking changes in this pr and it will not be merged to master

Reply:
@austincunningham I see the problem but if we change the package name and we are point it to another repo (next comment) there is not sense to me to have it. Also I can't see why build the library to 4.2 should not be merge to master.

Reply:
@davidffrench what do you think ?

Reply:
@danielpassos The decision was made by product management to merge this into a separate branch for the customer to use. The reason for not merging into master was due to no new features for RHMAP since it is in phase 2 maintenance support.

These changes are not getting published to cocoapods.